### DISCUSSION TITLE
How to set properties to use in workflows

### DISCUSSION THREAD

**Discussion by shekarpcs** [Netflix/conductor#2749] [General]
We have some values which may change such as URLs, error codes etc. 

How to make these configurable and use in Workflow? we have using 3.x version. 

can we pass these through config properties and use in workflow?

**Comment by aravindanr** [comment]
That's possible. You can check out these two links,

https://netflix.github.io/conductor/configuration/workflowdef/#wiring-inputs-and-outputs
https://netflix.github.io/conductor/labs/beginner/

both of which include examples how properties can be wired in the input and output

**Reply by shekarpcs** [reply]
Arav, 
I was trying to read them through conductor-config.properties or some -D parameters.

is there any way we can do that?

**Reply by apanicker-nflx** [reply]
Can you provide more info about how you intend to use these properties? This would help us understand the use case better and provide clearer guidance.

**Reply by shekarpcs** [reply]
We need to pass configurable URL to workflow  based on environment

**Comment by shekarpcs** [comment]
I figured it out . We can pass -DAPIURL=value at command line 
Above property can be referred in the workflow as ${APIURL}
